src/core1_task.rs

```diff
@@ -574,6 +580,16 @@ pub fn core_1_task(
     };
 
     let is_cptv = flash_storage.has_cptv_files(false);
+    let mut made_startup_status_recording = is_cptv && has_files_to_offload;
```

Comment:
If `is_cptv` is `true`, can `has_files_to_offload` ever be `false`?

Reply:
nope.